packages/scripts/src/monitor-dvm/README.md

```diff
@@ -0,0 +1,31 @@
+# DVMv2 Monitoring
+
+## Live Monitor Bots
+
+DVMv2 monitor bots can follow mainnet and goerli networks to report on real time events based on the provided configuration.
+
+The main entry point to live monitor bots is running:
+
+```
+yarn ts-node packages/scripts/src/monitor-dvm/index.ts
+```
+
+All the configuration should be provided with following environment variables:
+
+- `CUSTOM_NODE_URL` is L1 network node endpoint (Ethereum mainnet or goerli).
+- `POLLING_DELAY` is value in seconds for delay between consecutive runs, defaults to 1 minute. If set to 0 then running in serverless mode will exit after the loop.
+- `STARTING_BLOCK_NUMBER` and `ENDING_BLOCK_NUMBER` defines block range to look for events on L1 network. These are mandatory when `POLLING_DELAY=0`.
+- `UNSTAKES_ENABLED` is boolean enabling/disabling monitoring large unstake attempts (disabled by default).
+- `UNSTAKE_THRESHOLD` is UMA amount threshold to use when `UNSTAKES_ENABLED=true` (defaults to `"0"`).
```

Comment:
what units?

Reply:
fixed in 201b24ce20ceff01c388ca78ee8f1ce72772bd9f